docs/plugins.md

```diff
@@ -87,7 +87,7 @@ These plugins apply transformations to your code.
 - [es2015-block-scoping](/docs/plugins/transform-es2015-block-scoping/)
 - [es2015-classes](/docs/plugins/transform-es2015-classes/)
 - [es2015-computed-properties](/docs/plugins/transform-es2015-computed-properties/)
-- [es2015-constants](/docs/plugins/check-es2015-constants/)
```

Comment:
we already link to block scoping.

> [es2015-block-scoping](/docs/plugins/transform-es2015-block-scoping/)

the const checking was moved to the plugin that is already there.